packages/web/src/pages/sign-on/components/ProfileForm.tsx

```diff
@@ -185,14 +225,13 @@ const ProfileForm = (props) => {
           <Spring
             from={{ opacity: 0 }}
             to={{ opacity: 1 }}
-            leave={{ opacity: 0 }}
             config={{ duration: 200 }}
           >
             {(animProps) => (
               <div style={animProps} className={styles.suggestTwitter}>
                 <InstagramAuth
-                  onFailure={(...args) => console.log(args)}
-                  onSuccess={props.onInstagramLogin}
+                  onFailure={console.log}
```

Comment:
Just checking this is what we want

Reply:
Not super critical, I did this to appease TS. But yeah it should probably be passing all the args as one arg to console.log